Cargo.lock

```diff
@@ -197,7 +197,6 @@ dependencies = [
  "quote",
  "semver",
  "typed-arena",
```

Comment:
If this is the last use case for this crate, we need to remove it from the root `Cargo.toml` as well.
Not sure why it wasn't reported by `rustc` already :(